### DISCUSSION TITLE
Players do not receive claim blocks from any of my crates

### DISCUSSION THREAD

**Discussion by LewisRad** [GriefPrevention/GriefPrevention#1419] [Q&A] [answered]
Hello, 

I am running a server and some of my crates have a chance to win claim blocks. So using the command /acb %player% amount. but for some reason, only operators are able to receive the claim blocks from the crate. It is saying that other players do not have permission. Is there a fix to this? My crates plugin is GoldenCrates. This has worked before, but ever since I made it possible for players to buy claimblocks, this has caused the above issue.

Thanks!

**Comment by bigpresh** [comment]
It sounds like it's running the command as the user, rather than as the console (and naturally users don't have the ability to give themselves bonus claim blocks).  I'm not familiar with GoldenCrates so I don't know if you can change it.

**Comment by bigpresh** [comment, marked as answer]
From a quick Google, looking at a sample config: https://github.com/nulli0n/GoldenCrates/blob/master/GoldenCrates/src/crates/mystery.yml

... it shows commands like `- '[CONSOLE] eco give %player% 10'` which looks to be run as the console rather than as the user receiving it - maybe that would help?
